src/main/java/com/playmonumenta/libraryofsouls/SoulEntry.java

```diff
@@ -143,12 +138,12 @@ public Map<Soul, Double> getAverageSouls() {
 	}
 
 	@Override
-	public Double getWidth() {
+	public @Nullable Double getWidth() {
```

Comment:
Can you clarify why these are nullable? I suppose legacy entries could have null, but I made a command to automatically populate these values for the entire database, so everything should have them now except legacy entries.

Reply:
I don't like that these are nullable either. I tried not making them nullable, but that requires reworking the logic a bit. It probably would be nice to do that? But this matches existing behavior so I think it should be additional work. Not sure where I'd even put a comment about this, given that these getWidth() / getHeight() things are in a bunch of files.